test/swath-projector/utilities.py

```diff
@@ -0,0 +1,78 @@
+""" A module containing utility functionality used by the Swath Projector
```

Comment:
I've popped a separate `utilities.py` file in each test subdirectory. It's a bit repetitive, but there are some slight differences between the files.